packages/integrations/deno/README.md

```diff
@@ -1,176 +1,3 @@
-
-```bash
-npm run preview
-```
-
-## Usage
-
-After [performing a build](https://docs.astro.build/en/guides/deploy/#building-your-site-locally) there will be a `dist/server/entry.mjs` module. You can start a server by importing this module in your Deno app:
-
-```js
-import './dist/server/entry.mjs';
-```
-
-See the `start` option below for how you can have more control over starting the Astro server.
-
-You can also run the script directly using deno:
-
-```sh
-deno run --allow-net --allow-read --allow-env ./dist/server/entry.mjs
-```
-
-## Configuration
-
-To configure this adapter, pass an object to the `deno()` function call in `astro.config.mjs`.
-
-```js
-// astro.config.mjs
-import { defineConfig } from 'astro/config';
-import deno from '@astrojs/deno';
-
-export default defineConfig({
-  output: 'server',
-  adapter: deno({
-    //options go here
-  }),
-});
-```
-
-### start
-
-This adapter automatically starts a server when it is imported. You can turn this off with the `start` option:
-
-```js
-import { defineConfig } from 'astro/config';
-import deno from '@astrojs/deno';
-
-export default defineConfig({
-  output: 'server',
-  adapter: deno({
-    start: false,
-  }),
-});
-```
-
-If you disable this, you need to write your own Deno web server. Import and call `handle` from the generated entry script to render requests:
-
-```ts
-import { serve } from 'https://deno.land/std@0.167.0/http/server.ts';
-import { handle } from './dist/server/entry.mjs';
-
-serve((req: Request) => {
-  // Check the request, maybe do static file handling here.
-
-  return handle(req);
-});
-```
-
-### port and hostname
-
-You can set the port (default: `8085`) and hostname (default: `0.0.0.0`) for the deno server to use. If `start` is false, this has no effect; your own server must configure the port and hostname.
-
-```js
-import { defineConfig } from 'astro/config';
-import deno from '@astrojs/deno';
-
-export default defineConfig({
-  output: 'server',
-  adapter: deno({
-    port: 8081,
-    hostname: 'myhost',
-  }),
-});
-```
-
-## Examples
-
-The [Astro Deno](https://github.com/withastro/astro/tree/main/examples/deno) example includes a `preview` command that runs the entry script directly. Run `npm run build` then `npm run preview` to run the production deno server.
-
-## Troubleshooting
-
-For help, check out the `#support` channel on [Discord](https://astro.build/chat). Our friendly Support Squad members are here to help!
-
-You can also check our [Astro Integration Documentation][astro-integration] for more on integrations.
-
-## Contributing
-
-This package is maintained by Astro's Core team. You're welcome to submit an issue or PR!
-
-## Changelog
-
-See [CHANGELOG.md](CHANGELOG.md) for a history of changes to this integration.
-
-[astro-integration]: https://docs.astro.build/en/guides/integrations-guide/
+The Deno adapter has a new home as part of the [Deno organization](https://github.com/denoland/deno-astro-adapter).
```

Comment:
```suggestion
This adapter is no longer maintained by Astro. Please see [the new repository for the Deno adapter](https://github.com/withastro/netlify-adapter) which is now maintained by the Deno organization.

[Instructions for removing this adapter and migrating to the new one](https://docs.astro.build/en/guides/integrations-guide/deno/) can be found in the Astro docs.
```
Just making link text as explicit as possible, and linking to our docs page which has instructions for updating your project.